packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -340,9 +347,18 @@ class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
     };
 
     focus: (number) => boolean = (i) => {
-        // $FlowFixMe[incompatible-use]
-        // $FlowFixMe[prop-missing]
-        ReactDOM.findDOMNode(this.refs["radio" + (i || 0)]).focus(); // eslint-disable-line react/no-string-refs
+        const ref = this.choiceRefs[i || 0];
```

Comment:
One of the big pieces when switching Choice to a functional component was that we could no longer stick a ref directly onto it. We still keep these refs in an array (now called choiceRefs), but we forward the refs down to the WB button; either:

```
BaseRadio -> Choice -> Clickable
or
BaseRadio -> ChoiceNoneAbove -> Choice -> Clickable
```

React's forwardRef combined with Flow's typing makes it kind of hard to read, but I like that it's more explicit now. Thanks Kevin for all your help with that.

Reply:
So in either case the underlying ref will be the `Clickable`, is that right?

Reply:
Would love to pair sometime to learn more about refs and how they are affected by switching from class to functional. :)

Reply:
@kevinbarabash right, I don't see how there's any chance for it to not be a Clickable. Just sometimes it's further down the line than others.

@Myranae any time!